Refine docs (#505)
pingcap/chaos-mesh: doc/time_chaos.md

## Limitation

> * Time modification only can be injected into the main process of container.

Reviewer: ```suggestion
* Time modification can only be injected into the main process of container.
```

* Time chaos has no effect on pure syscall `clock_gettime`.

> * All injected vdso call use pure syscall to get realtime, so clock-related function call becomes much slower.

Reviewer: ```suggestion
* All injected vDSO calls use pure system calls to get realtime, so clock-related function call can be much slower.
```

Reviewer: Do we need to explain vDSO?  Also what do we mean by realtime?
